http/src/main/scala/org/http4s/blaze/http/http2/bits.scala

```diff
@@ -15,7 +15,7 @@ private[http2] object bits {
 
   object Masks {
     val INT31: Int = 0x7fffffff
-    val INT32: Long = 0xffffffffL
+    val INT32: Long = 0XFFFFFFFFL
```

Comment:
It's curious it doesn't capitalize the `Int` literals, too.

Reply:
Yea, that's weird.